PhysicsTools/PatAlgos/python/slimming/customizeMiniAOD_80Xto91XreMiniAOD_BTV.py

```diff
@@ -0,0 +1,15 @@
+import FWCore.ParameterSet.Config as cms
```

Comment:
I think that the "current release" label should be removed from the file name.
Unless the number of customizations becomes huge, I'd rather keep it all in one file.

How about customizeMiniAOD_80XLegacyInput.py